README.adoc

```diff
@@ -231,7 +231,7 @@ The executables are built using the <<Node / JavaScript Setup,Node / JavaScript>
 * Copy or clone the reveal.js presentation framework in the directory where you will build your slidedeck.
   Here we do a shallow clone of the repo:
 
-  $ git clone -b 3.9.2 --depth 1 https://github.com/hakimel/reveal.js.git
+  $ git clone -b 3.7.0 --depth 1 https://github.com/hakimel/reveal.js.git
```

Comment:
That should not be here. Please revert.

Reply:
Ooops, it looks like I messed up my rebase 😅

Reply:
```suggestion
  $ git clone -b 3.9.2 --depth 1 https://github.com/hakimel/reveal.js.git
```